src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -25,22 +25,19 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap101aot_Debug|AnyCPU'" />  
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap101aot_Release|AnyCPU'" />  
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap101aot_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap101aot_Release|AnyCPU'" />
   <ItemGroup Condition="'$(TargetGroup)'==''">
     <TargetingPackReference Include="System.Private.CoreLib" />
     <ProjectReference Include="$(SourceDir)\mscorlib.WinRT-Facade\mscorlib.WinRT-Facade.csproj" />
     <!-- Needed for the compiler to resolve IObservableMap.MapChanged. -->
     <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
-    <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj">
-       <Aliases>System_Runtime_Extensions</Aliases>
+    <ProjectReference Include="..\..\System.Runtime.Extensions\ref\System.Runtime.Extensions.csproj">
```

Comment:
I believe he had to do it like this so that we find MemoryStream at compile time. If we use the implementation, then MemoryStream would be type-forwarded to CoreLib which is ok in CoreCLR but not in UWP since MemoryStream is not there yet. That is why we instead are referencing the ref in here. If we wanted to reference the ref in all P2Ps then we would need aliases, since System.Runtime and CoreLib would have type clashes.

---
In reply to: [88064177](https://github.com/dotnet/corefx/pull/13659#discussion_r88064177) [](ancestors = 88064177)